response_operations_ui/controllers/party_controller.py

```diff
@@ -125,6 +130,48 @@ def search_respondent_by_email(email):
     return response.json()
 
 
+def search_respondents(first_name, last_name, email_address, page=0):
+    if bool(os.environ.get('FAKE_PARTY_DATA', False)):
+        logger.debug('Respondent Search: Using fake data for controller')
+        data = mock_search_respondents(first_name, last_name, email_address, page)
+    else:
+        url = _get_search_respondents_url(first_name=first_name,
+                                          last_name=last_name,
+                                          email_address=email_address,
+                                          page=page)
+        response = requests.get(url, auth=app.config['PARTY_AUTH'])
+        if response.status_code != 200:
+            raise SearchRespondentsException(response,
+                                             first_name=first_name,
+                                             last_name=last_name,
+                                             email_address=email_address,
+                                             page=page)
+        data = response.json()
+
+    return data
+
+
+def _get_search_respondents_url(**kwargs):
```

Comment:
2 things.  This should probably be called `_get_search_respondents_url` second.  requests.get has a params argument that should probably be used here instead of a function to insert the params into the url by hand

Reply:
> This should probably be called `_get_search_respondents_url`
I'm confused.   That _is_ what it's called.

Regarding the keywords, do you feel particularly strongly about it?  I'd like to move this on quickly if poss, and this works.

Reply:
I copied it then forgot to change it.... I was going to suggest `_build_search_respondents_url` because it is building it

Reply:
And for the 2nd thing, I get you want to move quickly, but with this we have another function that we have to maintain, and fix problems with if we find any; meanwhile, we have a tool that's been tested a ton and definitely works.
Less code means there's less stuff that can break

Reply:
So, you want to remove the function entirely?

Reply:
Yes. Just create a list of parameters (I think request.get takes a list of params as an arguement... or maybe it's a dict...) and pass that into the request.get call.  It'll bloat that function a tiny bit but it won't be much.

Reply:
Done. YAGNI